config.yaml

```diff
@@ -18,7 +18,7 @@ options:
     default: ""
     type: string
     description: |
-      LicenseKey should be a file, appiled as a bas64 string
+      LicenseKey should be a file, applied as a bas64 string
```

Comment:
```suggestion
      LicenseKey should be a file, applied as a base64 string
```

Reply:
:egg: on my face!!  i should've done `s/$spelling_error/$spelling_fix/g` instead of just stopping that the first one ;)

in my defense, i feel like `codespell` should catch that...